src/model/RaceModel.js

```diff
@@ -0,0 +1,13 @@
+class RaceModel {
+  constructor(cars) {
+    this.cars = cars;
+  }
+
+  getWinner() {
+    const longestDistance = Math.max(...this.cars.map((car) => car.distance));
+    return this.cars
+      .filter((car) => car.distance === longestDistance)
+      .map((car) => car.name);
+  }
+}
+export default RaceModel;
```

Comment:
RaceModel도 목적에 맞게 잘 설계된 것 같아요 여기서도 약간의 코드 수정이 있으면 더 관심사 분리를 지킬 수 있을 것 같아요!

```suggestion
class RaceModel {
  constructor(cars) {
    this.cars = [...cars]; 
  }

  getLongestDistance() {
    if (this.cars.length === 0) return 0; 
    return Math.max(...this.cars.map((car) => car.distance));
  }

  getWinner() {
    const longestDistance = this.getLongestDistance();
    return this.cars
      .filter((car) => car.distance === longestDistance)
      .map((car) => car.name);
  }
}

export default RaceModel;
```

Reply:
이렇게 더 쪼갤 수 있는 방법이 있군요
감사합니다~~!!